### PR TITLE
Update update-daemon-set.md

### PR DESCRIPTION
**Pull request by MdSahil-oss**
<h5>What type of PR is this ?</h5>
Improve documentation.

<h5>Which type of issue this PR fixes</h5>
fixes 31311(issue).

### CODE REVIEW

File: content/en/docs/tasks/manage-daemon/update-daemon-set.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ You may want to set
 (default to 1),
 [`.spec.minReadySeconds`](/docs/concepts/workloads/controllers/deployment/#min-ready-seconds) 
 (default to 0) and 
-[`.spec.maxSurge`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#max-surge) 
+[`.spec.updateStrategy.rollingUpdate.maxSurge`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#max-surge)
```
Comment (mehabhalodiya) [inline content/en/docs/tasks/manage-daemon/update-daemon-set.md:42]:
```suggestion
[`.spec.updateStrategy.rollingUpdate.maxSurge`](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/daemon-set-v1/#DaemonSetSpec) 
```

**Review by mehabhalodiya — COMMENTED** [review]
I would recommend you to change, considering [Tim's comment](https://github.com/kubernetes/website/issues/31311#issuecomment-1012041310).

Thank you.

**Comment by MdSahil-oss** [comment]
@mehabhalodiya I have changed update-daemon-set.md as per [Tim's comment](https://github.com/kubernetes/website/issues/31311#issuecomment-1012041310)

**Review by mehabhalodiya — COMMENTED** [review]
LGTM!
@MdSahil-oss congrats on your first PR 🎉 
Keep contributing more 🚀

**Comment by mehabhalodiya** [comment]
Awaiting another sig docs reviewer/approver to sign off these changes.

File: content/en/docs/tasks/manage-daemon/update-daemon-set.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ You may want to set
 (default to 1),
 [`.spec.minReadySeconds`](/docs/concepts/workloads/controllers/deployment/#min-ready-seconds) 
 (default to 0) and 
-[`.spec.maxSurge`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#max-surge) 
+[`.spec.updateStrategy.rollingUpdate.maxSurge`](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/daemon-set-v1/#DaemonSetSpec)
```
Comment (sftim) [inline content/en/docs/tasks/manage-daemon/update-daemon-set.md:42]: It's not a regression, but this should really be:
```suggestion
[`.spec.updateStrategy.rollingUpdate.maxSurge`](/docs/reference/kubernetes-api/workload-resources/daemon-set-v1/#DaemonSetSpec)
```

**Review by sftim — COMMENTED** [review]
/approve

**Comment by sftim** [comment]
Ah, a merge conflict? I can see the issue. It's not an actual conflict though.
/approve cancel

@MdSahil-oss do you know how to rebase? If so, please do. If not; please let SIG Docs know so we can suggest a different way forward.

**Comment by MdSahil-oss** [comment]
@sftim Sorry I'm beginner and don't know how to rebase Please suggest me another way

**Comment by mehabhalodiya** [comment]
We couldn't merge this as-is as there is a conflict that the automatic system does not know how to resolve.

@MdSahil-oss  if you haven't done a rebase before, you really have 2 options

1. Learn about Git rebasing. It's a great skill if you often work with source control; on the other hand, if you're mostly a writer or focus your skills in other areas than Git, it's probably not worth the effort (it's genuinely not that easy to learn the first time around).
2. Close this pull request, and open a new one based on the main branch that makes the same changes. In the new PR, mention that it's a replacement for PR #31354 

Here are some references/tips for the following:
- [merge conflicts and rebasing](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#merge-conflicts-and-rebasing)
- [squashing commits](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits)

**Comment by MdSahil-oss** [comment]
@mehabhalodiya made a lot of mistakes so let me open a new PR in replacement of this one

**Pull request closed without merging**